### PR TITLE
Refactoring Action classes to include Prompt Class in prompting the user

### PR DESCRIPTION
**Pull request by SafinazElsayed**
GuessLetterAction and SolvePuzzleACtion calling the PromptAction  method from the new Prompt class

### CODE REVIEW

**Review by cychoi00 — APPROVED** [review]
I think the PromptAction method may need to be compared with some of the other Prompt methods, but approving for now. Also the Prompt.PromptAction() is currently being called in the Execute methods, but they are expected to be updated in the future.

File: WheelOfFortune/Prompt.cs [hunk 1]
```diff
@@ -98,6 +98,15 @@ public static char GetActionType()
...
         {
+            if (action.ActionTypeProperty == Action.ActionType.GuessLetterAction)
+            {
+                Console.WriteLine("Start Guessing: Enter a letter:");
+            }
+            else if(action.ActionTypeProperty == Action.ActionType.SolvePuzzleAction)
+            {
+                Console.WriteLine("What is your solution to the puzzle?");
+            }
+
```
Comment (cychoi00) [inline WheelOfFortune/Prompt.cs:109]: Here, we should also have logic to validate and read the inputs from the user

Comment (SafinazElsayed) [reply WheelOfFortune/Prompt.cs:109]: yes I have it locally and will push in different PR..the Prompt.PromptAction will be called fromStartTuen in a coming PR as well

**Comment by SafinazElsayed** [comment]
Refactoring the Action classess to add the prompt action initial implementation

**Pull request merged**